src/Client.h

```diff
@@ -318,6 +381,11 @@ class Client : public ClientBase{
 		//send
 		for (unsigned i = 0; i < g_pApp->m_const_params.burst_size && !g_b_exit; i++) {
 			client_send_packet(ifd);
+#ifdef  USING_VMA_EXTRA_API
+			if (g_pApp->m_const_params.fd_handler_type == VMAPOLL && g_vma_api && !g_pApp->m_const_params.b_client_ping_pong){
+				m_ioHandler.waitArrival();
```

Comment:
Please consider checking `g_vma_api` as first in all places because other stuff can not be workable if `g_vma_api == NULL`.